torch/onnx/symbolic_helper.py

```diff
@@ -376,7 +376,7 @@ def _is_all_integral(scalars):
     # infer input types from dtype. If not, then check if any of start, stop,
     # or step are floating point, and infer the type from get_default.
     # Otherwise, the dtype is inferred to be torch.int64.
-    if _is_value(dtype) and _is_none(dtype):
+    if not _is_value(dtype) and dtype is None:
```

Comment:
I'm not sure why check for _is_value needed, if dtype is not None it will go to else block

Reply:
Fixed this, the _get_arange_dtype helper produces a torch.C._Value, for cases in which dtype is not provided, hence the check